tests/unit/src/test/resources/toplevelwithinner-scala3/example/Ord.scala

```diff
@@ -0,0 +1,12 @@
+package example
+
+trait Ord/*example.Ord#*/[T]:
+  def compare(x: T, y: T): Int
+
+given/*example.Ord$package.*/ intOrd: Ord[Int] with
```

Comment:
```suggestion
given intOrd: Ord[Int] with
```
This looks suspicious, might not be related to the changes, but let's raise an issue just in case

Reply:
Yes, it looks a bit odd but it is as expected.